### PR TITLE
fix(vulns): ids should be numbers

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

File: vuln/npm/387.json [hunk 1]
```diff
@@ -18,4 +18,4 @@
   "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:N/A:N",
   "cvss_score": 8.6,
   "coordinating_vendor": ""
-}
+}
```
Comment (ChALkeR) [inline vuln/npm/387.json]: Nit: why removing newline at the end of file?

Comment (lirantal) [reply vuln/npm/387.json]: ugh, editor config :(
we should probably set up some linting for the project

Comment (ChALkeR) [reply vuln/npm/387.json]: @lirantal There is already linting, run `npm it`. It could to be improved, though.

**Comment by ChALkeR** [comment]
@lirantal Now it adds a second trailing newline…

**Pull request merged**